### ISSUE TITLE
Abnormal circulating keto acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6018]
**Preferred term label:** Abnormal circulating keto acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Any deviation form the normal concentration of a keto acids in the blood circulation.

**Parent term (use hpo.jax.org/app)** Abnormal circulating metabolite concentration HP:0032180

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated circulating 3-methyl-2-oxovaleric acid concentration

**Synonyms** Elevated 2-keto-3-methylvaleric acid

**Definition (free text, please give PubMed ID)** Concentration of 3-methyl-2-oxovaleric acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating keto acid concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Abnormal circulating alpha-ketoisocaproic acid concentration

**Synonyms** 

**Definition (free text, please give PubMed ID)** Concentration of alpha-ketoisocaproic acid in the blood is outside the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating keto acid concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating alpha-ketoisocaproic acid concentration

**Synonyms** Decreased circulating 2-ketoisocaproic acid concentration

**Definition (free text, please give PubMed ID)** Concentration of alpha-ketoisocaproic acid in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating alpha-ketoisocaproic acid concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating alpha-ketoisocaproic acid concentration

**Synonyms** Increased circulating 2-ketoisocaproic acid concentration

**Definition (free text, please give PubMed ID)** Concentration of alpha-ketoisocaproic acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating alpha-ketoisocaproic acid concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated urine alpha-ketoisocaproic acid concentration 

**Synonyms** Elevated urine 2-ketoisocaproic acid concentration

**Definition (free text, please give PubMed ID)** Concentration of  alpha-ketoisocaproic acid in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal keto acid urine concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
